roles/mapping/tasks/main.yml

```diff
@@ -7,10 +7,10 @@
   tags:
     - aur
 
-- name: Install OpenCPN
-  aur: name=opencpn user={{ user.name }}
-  tags:
-    - aur
+#- name: Install OpenCPN
```

Comment:
Good point.  OpenCPN is one of my primary use cases, and I found various forks and varying compilation arg suggestions.. planning to play around with it before making a 'recommendation' into this role.  Let me update to comment as such.